### PR TITLE
Feature - Api for search correct - Search screen

### PR DESCRIPTION
**Pull request by at-tanhuynh**

### CODE REVIEW

**Comment by at-lamle** [comment]
:+1: There are no linter violations.

File: MyApp/View/Base/LocationManager.swift [hunk 1]
```diff
@@ -65,4 +79,9 @@ extension LocationManager: CLLocationManagerDelegate {
             break
         }
     }
+
```
Comment (at-lamle) [inline MyApp/View/Base/LocationManager.swift]: WARNING - Lines should not have trailing whitespace.

**Comment by at-lamle** [comment]
Total linter violations in pull request: 1
Serious: 0
Warnings: 1

**Comment by at-lamle** [comment]
:+1: There are no linter violations.

File: MyApp/View/Base/LocationManager.swift [hunk 2]
```diff
@@ -50,6 +60,10 @@ final class LocationManager: NSObject {
     func stopStandardLocationService() {
         clLocationManager.stopUpdatingLocation()
     }
+
+    func getCurrentLocation() -> CLLocationCoordinate2D? {
```
Comment (at-hainguyen3) [inline MyApp/View/Base/LocationManager.swift]: Không dùng thì xoá luôn

File: MyApp/View/Controller/Search/SearchCorrect/SearchCorrectViewController.swift [hunk 3]
```diff
@@ -7,25 +7,40 @@
...
+
+protocol SearchCorrectViewControllerDelegate: class {
+    func controller(_ controller: SearchCorrectViewController, needsPerform action: SearchCorrectViewController.Action)
+}
 
 final class SearchCorrectViewController: UIViewController {
 
+    // MARK: - Enum
+    enum Action {
+        case passRestaurantIdToDetail(id: String)
```
Comment (at-hainguyen3) [inline MyApp/View/Controller/Search/SearchCorrect/SearchCorrectViewController.swift]: Rename

Comment (at-hainguyen3) [reply MyApp/View/Controller/Search/SearchCorrect/SearchCorrectViewController.swift]: có thể đặt `tapPin`

**Pull request merged**